node/chainSimulator/components/testOnlyProcessingNode_test.go

```diff
@@ -76,6 +76,21 @@ func TestNewTestOnlyProcessingNode(t *testing.T) {
 		newHeader, err := node.ProcessComponentsHolder.BlockProcessor().CreateNewHeader(1, 1)
 		assert.Nil(t, err)
 
+		rootHash, err := node.GetStateComponents().AccountsAdapter().RootHash()
+		if err != nil {
+			log.Error("SetRootHashOfGenesisBlocks", "err", err)
```

Comment:
Perhaps smth like this?
```suggestion
			log.Error("node.GetStateComponents().AccountsAdapter().RootHash()", "err", err)
```

Reply:
yes, fixed it